pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -263,7 +263,7 @@ private static void SkylineBeginInvoke(Action act)
             }
         }
 
-        protected static void RunDlg<TDlg>(Action show, [InstantHandle] Action<TDlg> act = null, bool pause = false, int millis = -1) where TDlg : Form
+        protected static void RunDlg<TDlg>([InstantHandle] Action show, [InstantHandle] Action<TDlg> act = null, bool pause = false, int millis = -1) where TDlg : Form
```

Comment:
Just curious - what's this about?

Reply:
It lets ReSharper know that the Action "show" is not going to be invoked after RunDlg returns.
Without it, ReSharper would have given me the "access to modified closure" warning because the "transitionSettingsUi" variable that I was using inside of my lambda got modified later on in the DoTest method.